File: Beginner/Interval_1037/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double number = sc.nextDouble();
        String message;

        if (number >= 0 && number <= 25) {
            message = "Intervalo [0,25]";
        } else if (number > 25 && number <= 50) {
            message = "Intervalo (25,50]";
        } else if (number > 50 && number <= 75) {
            message = "Intervalo (50,75]";
        } else if (number > 75 && number <= 100) {
            message = "Intervalo (75,100]";
        } else {
            message = "Fora de intervalo";
        }
        System.out.println(message);
        sc.close();
    }
}

File: Beginner/Interval_1037/main.py
number = float(input())
message = ""

if number >= 0 and number <= 25:
    message = "Intervalo [0,25]"
elif number > 25 and number <= 50:
    message = "Intervalo (25,50]"
elif number > 50 and number <= 75:
    message = "Intervalo (50, 75]"
elif number > 75 and number <= 100:
    message = "Intervalo (75, 100]"
else:
    message = "Fora de intervalo"

print(message)
